README.md

```diff
@@ -101,6 +103,28 @@ You need to setup a URL that will listen for interactive message callbacks. Look
 
 You need to setup a URL that will listen for commands. Look at the usage example to have a better understanding.
 
+### Threads
+
+Messages now include a `messageid` in their conversation id, this new parameter mimics what MS Teams does to handle threads there. It looks something like `BXXX:TXXX:CXXX;messageid=123456`.
```

Comment:
By default, replies to user's messages are threaded. This means that if you want to have an unthreaded conversation with a user you will need to manually change the address. This mimics what Microsoft Teams does. A conversation id will look like this `BXXX:TXXX:CXXX;messageid=123456`.